windows/lib/WixSDK.js

```diff
@@ -257,20 +257,14 @@ function(app_path, xwalk_path, meta_data, callback) {
         });
     }
 
-    // Extensions are supposed to be in the source application root dir: app_path.
-    // Then we copy them to the sub-directory of the installer folder:
-    //     app_files_folder/xwalk-extensions
-    // So, if we still fully copy the source application root directory, all the
-    // extensions will be duplicated.
-    // Extensions can be divided by categories in seperate directories.
+    // Only .dll files in extension folders will be picked up into
+    // app_files_folder/xwalk-extensions.
     this._manifest.extensions.forEach(function(extDir) {
         installExtensionDlls(extDir, app_extensions_folder);
-        if (path.normalize(path.dirname(extDir)) == path.normalize(app_path)) {
-            extensions_relative_dir.push(path.relative(app_path, extDir));
-        }
     }.bind(this));
 
     // Skip in-folder extensions copying to avoid duplication.
+    // Because the needed extension files has already be installed by previous step.
     installFiles(app_path, app_files_folder, this._manifest.extensions);

```

Comment:
@r0b5t4 We have skipped the in-folder extensions when install the web staff into the installer. So there will be no dup.